lib/tasks/docker.rake

```diff
@@ -33,9 +35,16 @@ end
 
 desc 'Run JS and Ruby linters'
 task 'docker:lint' do
-  success = run_or_fail("bundle exec rubocop --parallel")
-  success = run_or_fail("eslint app/assets/javascripts test/javascripts")
-  success = run_or_fail("eslint --ext .es6 app/assets/javascripts test/javascripts plugins")
+  success = true
+
+  if ENV["SINGLE_PLUGIN"]
```

Comment:
It's been SINGLE_PLUGIN for the past month (since https://github.com/discourse/discourse/pull/4973). Can change it if absolutely necessary, but it will mean making the change in `docker:test`, adjusting travis config on a few plugins, and updating howto docs on Meta. Let me know 🙂